src/applications/post-911-gib-status/tests/containers/IntroPage.unit.spec.jsx

```diff
@@ -30,7 +30,8 @@ describe('<IntroPage/>', () => {
 
   it('should render a link to /status', () => {
     const wrapper = shallow(<IntroPage {...defaultProps}/>);
-    expect(wrapper.find('Link').first().props().to).to.equal('status');
+    const summary = wrapper.find('VetsDotGovSummary').dive();
+    expect(summary.find('Link').first().props().to).to.equal('status');
```

Comment:
I moved the Vets.gov introduction into its own component to remove some of the clutter, and had to adjust this afterwards.